web/config.py

```diff
@@ -226,6 +226,8 @@
 else:
     if SERVER_MODE:
         DATA_DIR = os.path.join('/var/lib/', APP_PATH)
+    elif os.path.exists("/.flatpak-info"):
```

Comment:
While installing, you can override this config setting into the config_local or config_system file, so you can remove it from here. Apart from this, other changes look good to me.

Reply:
Can I use system variables in there, like needed here, for `XDG_CONFIG_HOME`?

Reply:
config_distro.py is used of for package distribution related config. https://www.pgadmin.org/docs/pgadmin4/8.5/config_py.html#the-config-py-file.

Reply:
Is this done correctly? I don't think it works?

https://github.com/razzeee/org.pgadmin.pgadmin4/pull/2#issuecomment-2048260923

Reply:
> Is this done correctly? I don't think it works?
> 
> [razzeee/org.pgadmin.pgadmin4#2 (comment)](https://github.com/razzeee/org.pgadmin.pgadmin4/pull/2#issuecomment-2048260923)

Can you please specify, what you tried and what's not working?

Reply:
Tried the distro_config.py like this

![image](https://github.com/pgadmin-org/pgadmin4/assets/5943908/8bd7321f-340e-425a-8fc4-71e04bdc3842)


And it doesn't seem to stick/lead to the same path as it did with changing the data_dir in the config.py itself

Reply:
Can you check if there is any config_local or config_system in your env which is overriding it?
Where did you put the config_distro? It should be beside config.py. try debugging web/pgadmin/evaluate_config.py where it is read and merged.

Reply:
Thank you, I think I've figured this out now

Reply:
Removed this commit as it's handled in the config of the flatpak now